languages/elixir/exercises/concept/basic-strings/lib/high_school_sweetheart.ex

```diff
@@ -0,0 +1,32 @@
+defmodule HighSchoolSweetheart do
+  def first_letter(name) do
+    raise "Please implement the first_letter/1 function"
+  end
+
+  def initial(name) do
+    raise "Please implement the initial/1 function"
+  end
+
+  def initials(full_name) do
+    raise "Please implement the initials/1 function"
+  end
+
+  def pair(full_name1, full_name2) do
+    #         ******       ******
+    #       **      **   **      **
+    #     **         ** **         **
+    #    **            *            **
+    #    **                         **
+    #    **                         **
+    #     **                       **
+    #       **                   **
+    #         **               **
+    #           **           **
+    #             **       **
+    #               **   **
+    #                 ***
+    #                  *
```

Comment:
Is it helpful that the heart is already here as a comment? Does it need extra explanations?

I was having a lot of trouble copy-pasting it because my IDE has a lot of opinions about what to do with whitespace.

Reply:
I think it would be useful for us to have it as a pattern for people to use, since the goal is to get them to make it display the initials, not critique on their ascii-art.  @ErikSchierboom?

Reply:
Agreed. We're not trying to teach them how to do ASCII art (there's a nice practice exercise there!), but how to work with strings. Having it already in a comment makes total sense.